storage.go

```diff
@@ -239,6 +257,10 @@ func getCache(ctx context.Context, key string) (*Object, error) {
 }
 
 func purgeCache(ctx context.Context, key string) error {
+	if LocalCache != nil {
+		LocalCache.Remove(key)
```

Comment:
What happens when you have 2+ instances running on App Engine. Only one will receive purgeCache call and so other instances will keep the stale cache in memory, unfortunately.